homeassistant/util/yaml/loader.py

```diff
@@ -286,37 +285,37 @@ def _parse_yaml(
 
 @overload
 def _add_reference(
-    obj: list | NodeListClass,
-    loader: LoaderType,
-    node: yaml.nodes.Node,
+    obj: list | NodeListClass, loader: LoaderType, node: yaml.nodes.Node
 ) -> NodeListClass: ...
 
 
 @overload
 def _add_reference(
-    obj: str | NodeStrClass,
-    loader: LoaderType,
-    node: yaml.nodes.Node,
+    obj: str | NodeStrClass, loader: LoaderType, node: yaml.nodes.Node
 ) -> NodeStrClass: ...
 
 
 @overload
 def _add_reference(
-    obj: _DictT, loader: LoaderType, node: yaml.nodes.Node
-) -> _DictT: ...
```

Comment:
I'm guessing you dropped this because it never actually returned `_DictT` and would always be `NodeDictClass`?